monitor/core/src/main/java/core/JsonParsing.java

```diff
@@ -0,0 +1,131 @@
+package core;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+/**
+ * 
+ * @author Dionysis Athanasopoulos <dionysiscsuoi@gmail.com>
+ *
+ */
+
+public class JsonParsing {
+
+	private static void getArray( Object object ) throws ParseException {
```

Comment:
In my opinion, this kind of things difficult a lot the understanding of the code, if he want to return a list the proper way to do it is to initialize inside of the method, or returning a copy of the list.